x/swap/types/errors.go

```diff
@@ -4,8 +4,7 @@ import (
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 )
 
-// DONTCOVER
-
 var (
-	ErrCustom = sdkerrors.Register(ModuleName, 2, "")
+	ErrNotAllowed     = sdkerrors.Register(ModuleName, 1, "not allowed")
```

Comment:
```suggestion
	ErrNotAllowed     = sdkerrors.Register(ModuleName, 2, "not allowed")
```
Code 1 is reserved, custom error codes should start from 2

Reply:
good catch 👍